qiskit/algorithms/eigen_solvers/eigen_solver.py

```diff
@@ -92,14 +93,40 @@ def eigenstates(self, value: np.ndarray) -> None:
         self._eigenstates = value
 
     @property
-    def aux_operator_eigenvalues(self) -> Optional[List[ListOrDict[Tuple[complex, complex]]]]:
+    def aux_operator_values(self) -> Optional[List[ListOrDict[Tuple[complex, complex]]]]:
         """Return aux operator expectation values.
 
         These values are in fact tuples formatted as (mean, standard deviation).
         """
-        return self._aux_operator_eigenvalues
+        return self._aux_operator_values
+
+    @aux_operator_values.setter
+    def aux_operator_values(self, value: List[ListOrDict[Tuple[complex, complex]]]) -> None:
+        """set aux operator values"""
+        self._aux_operator_values = value
+
+    @property
+    @deprecate_function(
```

Comment:
Per the deprecation policy https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-policy (see 2a there) we can't deprecate this until 0.22.0 since we're adding the alternative API in this release. We can use a `PendingDeprecationWarning`, but we want to make sure there is a release that has both the old and new supported without warnings to make the migration for downstream libraries easier.

Reply:
Ah, my bad! Thanks for the info I'll update it 👍🏻